src/state/actions/watched-videos.ts

```diff
@@ -0,0 +1,15 @@
+export const ADD_WATCHED_VIDEOS = 'add_watched_videos'
+
+export const addWatchedVideos = (url: string): GetUnlocked => {
```

Comment:
`GetUnlocked` name does not exist - let's fix the typing in this file with something like this
```ts
export const ADD_WATCHED_VIDEOS = 'add_watched_videos'

export type WatchedVideosAction =
    | { type: typeof ADD_WATCHED_VIDEOS, url: string }

export const addWatchedVideos = (url: string): WatchedVideosAction => {
    return {
        type: ADD_WATCHED_VIDEOS,
        url,
    }
}
```